es6-arrow-functions/es6-arrow-functions-quiz-notes.md

```diff
@@ -0,0 +1,94 @@
+# es6-arrow-functions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the syntax for defining an arrow function?
+  const variable = (parameter) => {
+  function block
+  }
+
+- When an arrow function's body is not surrounded in curly braces (_concise body syntax_), what changes in its functionality?
+  Does not require a return operator
+
+- When using _concise body syntax_, how do you return an object literal?
+  Surround the object literal with parantheses
+
+- In the expression
+
+  ```js
+  foo(() => 42);
+  ```
+
+  - Identify the arrow function
+    () => 42
+
+  - How many arguments does the arrow function take?
+    0
+
+  - What value does it return?
+    42
+
+  - How many arguments are passed to the function `foo`?
+    1
+
+  - What type of argument is passed to the function `foo`?
+    function
+
+- In the expression
+
+  ```js
+  bar((y) => {
+    console.log(`4y = ${4 * y}`);
+  });
+  ```
+
+  - Identify the arrow function
+    (y) => {console.log(`4y = ${4 * y}`);}
+
+  - How many arguments does the arrow function take?
+    1
+
+  - What value does it return?
+    string
```

Comment:
It doesn't actually `return` anything! Well technically it does, but it would be `undefined`